tests/meson.build

```diff
@@ -117,6 +117,10 @@ if get_option('enable-nnfw-runtime')
   subdir('tizen_nnfw_runtime')
 endif
 
+if get_option('enable-edgetpu')
```

Comment:
Add `get_option('enable-edgetpu')` to the list of tensor filter which requires `test_models` to be installed along with them (at line 102)

Reply:
Fixed.